docs/BinSkimRules.md

```diff
@@ -914,3 +914,25 @@ This rule emits CSV data to the console for every compiler/language/version comb
 
 ---
 
+## Rule `BA6001.DisableIncrementalLinking`
+
+### Description
+
+Incremental linking support increases binary size and can reduce runtime performance. Fully optimized release builds should not specify incremental linking.
+
+### Messages
+
+#### `Pass`: Pass
+
+'{0}' was compiled with incremental linking disabled.
+
+#### `Warning`: Warning
+
+'{0}' was compiled with incremental linking enabled, which inflates binary size and can reduce performance.
+
+#### `NonReleaseBuild`: Pass
```

Comment:
>Pass [](http://example.com/codeflow?start=24&length=4)

So this should actually be emitted as a 'not applicable' message, i.e., this output doesn't appear to be a release build, therefore enforcing this particular standard does not make sense. IOW, we reserve 'pass' for 'all conditions for satisfying this particular quality standard lined up and you hit the mark'.

Reply:
I had that originally and changed it to a pass before I sent the PR (although it was a few weeks ago now and I'm struggling to remember my full rationale for that).  I'll change it back.